addon-manager/impl/pom.xml

```diff
@@ -70,6 +70,17 @@
          <artifactId>projects-api</artifactId>
          <scope>provided</scope>
       </dependency>
+      <dependency>
+		<groupId>net.lingala.zip4j</groupId>
+		<artifactId>zip4j</artifactId>
+		<version>1.3.2</version>
+	  </dependency>
+	  <dependency>
+	    <groupId>org.apache.commons</groupId>
```

Comment:
What is the pros of using other addons instead of external libraries? I don't know how to check if some functionality is already provided by some addon